src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1701,7 +1701,7 @@ public Allele getAltAlleleWithHighestAlleleCount() {
         return getAlternateAlleles().stream()
                 .map(allele -> new Tuple<>(allele, getCalledChrCount(allele)))
                 .max((alleleAndCount1, alleleAndCount2) -> Integer.compare(alleleAndCount1.b, alleleAndCount2.b))
-                .get()
+                .orElse(null)
                 .a;
```

Comment:
@kasont Do you have time to fix this? The problem is that `orElse(null)` can return null, and you're dereferencing the result of that expression directly via the `.a`. You need to protect against dereference of null, and have the method return null instead.